From fcc7d8280b63d99f49b47ddac4709701ab5476b0 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 6 Jun 2014 12:47:25 -0400
Subject: [redhat] keys: don't copy stub_key.x509 over

We only need rhel*.x509 copied over, stub_key.x509 was accidentally being
copied over to the rpm SOURCE dir following a wildcard addition in commit
56c99bf02a8a622acf4de90e9553c5850dfae967 when we were adding extra
modsigning keys for kpatch and dup modules.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/Makefile b/redhat/Makefile
index 62a0b7d..50761ad 100644
--- a/redhat/Makefile
+++ b/redhat/Makefile
@@ -99,7 +99,7 @@ sources-rh: $(TARBALL)
  @(if [ ! -f keys/extra_certificates ]; then \
   touch keys/extra_certificates; \
  fi)
- @cp cpupower.* x509.genkey keys/*.x509 keys/extra_certificates Makefile.common kabi/check-kabi kabi/Module.kabi_{ppc64,s390x,x86_64} kabi/kabi_whitelist_{ppc64,s390x,x86_64} configs/kernel-*.config rhpkg/Makefile scripts/sign-modules keys/*.cer $(SOURCES)/
+ @cp cpupower.* x509.genkey keys/rhel*.x509 keys/extra_certificates Makefile.common kabi/check-kabi kabi/Module.kabi_{ppc64,s390x,x86_64} kabi/kabi_whitelist_{ppc64,s390x,x86_64} configs/kernel-*.config rhpkg/Makefile scripts/sign-modules keys/*.cer $(SOURCES)/
 
 rh-sources: setup-source rh-configs sources-rh
 
-- 
1.7.1